mantidimaging/gui/windows/spectrum_viewer/test/spectrum_test.py

```diff
@@ -20,11 +22,27 @@
 @start_qapplication
 class SpectrumROITest(unittest.TestCase):
 
+    def setUp(self) -> None:
+        self.roi = SensibleROI(10, 20, 30, 40)
+        self.spectrum_roi = SpectrumROI("test_roi", self.roi)
+
     def test_WHEN_initialise_THEN_pos_and_size_correct(self):
-        roi = SensibleROI(10, 20, 30, 40)
-        spectrum_roi = SpectrumROI("", roi)
-        self.assertEqual(spectrum_roi.getState()["pos"], Point(10, 20))
-        self.assertEqual(spectrum_roi.getState()["size"], Point(20, 20))
+        self.assertEqual(self.spectrum_roi.getState()["pos"], Point(10, 20))
+        self.assertEqual(self.spectrum_roi.getState()["size"], Point(20, 20))
+
+    def test_WHEN_colour_changed_THEN_roi_colour_is_set(self):
+        self._colour = (1, 2, 58, 255)
```

Comment:
This does not need to be an instance variable, could just be `colour = (1, 2, 58, 255)`

(I was confused for a bit because I though it was setting the `_colour` attribute in the `SpectrumROI`)